cmd/podman/root.go

```diff
@@ -451,6 +451,15 @@ func rootFlags(cmd *cobra.Command, podmanConfig *entities.PodmanConfig) {
 		}
 		podmanConfig.Remote = true
 	} else {
+		// The --module's are actually used and parsed in
+		// `registry.PodmanConfig()`.  But we also need to expose them
+		// as a flag here to a) make sure that rootflags are aware of
+		// this flag and b) to have shell completions.
+		var containersConfModules []string
+		moduleFlagName := "module"
+		pFlags.StringSliceVar(&containersConfModules, moduleFlagName, []string{}, "Load the containers.conf(5) module")
```

Comment:
containersConfModules is not used so just define the flag with pFlags.StringSlice(), also set nil as default there is no need to allocate empty slices for this